server/galaxyls/tests/files/complex_inputs_01_command.xml

```diff
@@ -0,0 +1,38 @@
+<command><![CDATA[
+
+## Auto-generated command section
+## TODO: please review and edit this section as needed
+
+## Inputs
+
+#if str( \$c1.c1_action ) == "a1":
+    ${1:TODO_argument} '\$c1.c1_a1_p1'
+    ${2:TODO_argument} \$c1.sec1.number
+#elif str( \$c1.c1_action ) == "a2":
+    ${3:TODO_argument} '\$c1.c1_a2_p1'
+    #if str( \$c1.c2.c2_action ) == "c2a1":
+        ${4:TODO_argument} '\$c1.c2.c2_a1_p1'
+        ${5:TODO_argument} \$c1.c2.sec_c2a1.sub_sec_c2a1.sub_p01
+        ${6:TODO_argument} \$c1.c2.sec_c2a1.sub_sec_c2a1.sub_p02
+        \$--sub_p03
```

Comment:
here is a mistake ... it should be simply `$c1.c2.sub_sec_c2a1.sub_p03`

Reply:
Yes, this is using the `argument` value directly. I will fix it. Thanks!